pkg/reconciler/pipelinerun/controller.go

```diff
@@ -93,11 +97,38 @@ func NewController(images pipeline.Images) func(context.Context, configmap.Watch
 			DeleteFunc: impl.Enqueue,
 		})
 
+		AddPipelineRun := func(obj interface{}) {
+			pr := obj.(*v1alpha1.PipelineRun)
+			c.Logger.Infof("Adding PipelineRun %s/%s", pr.Namespace, pr.Name)
```

Comment:
would be great if these log messages could say _what_ the PipelineRun is being added to (in this case it's to the queue for deleting the Runs after the TTL expires?)